packages/api/src/index.ts

```diff
@@ -2,11 +2,12 @@ import useInfiniteQuery from './useInfiniteQuery';
 import useMutation from './useMutation';
 import useQuery from './useQuery';
 
+export * as p2p_entity from './hooks/p2p/entity';
```

Comment:
This will be used as:
```
import { p2p_entity } from '@deriv/api'
...
p2p_entity.payment_method.p2p_payment_methods.useGet()
```

Reply:
IMO that's too long can't we make it smaller?

Reply:
Can you make this camelCase too?
https://github.com/binary-com/deriv-app/tree/master/docs/JavaScript

Reply:
Sure, I think we can condense and shorten it into (with camel case):
```
p2p.paymentMethods.useGet()
p2p.advertiserPaymentMethods.useGet()
```